### PR TITLE
Database

### PR DESCRIPTION
**Pull request by pvinchon**

### CODE REVIEW

File: src/scala/tiefvision-web/conf/ddl.sql [hunk 1]
```diff
@@ -1,24 +0,0 @@
--- Copyright (C) 2016 Pau Carré Cardona - All Rights Reserved
```
Comment (paucarre) [inline src/scala/tiefvision-web/conf/ddl.sql]: this is now somewhere else right?

File: doc/developer.md [hunk 2]
```diff
@@ -28,11 +28,12 @@ git clone git@github.com:paucarre/tiefvision.git
...
-## H2 Database setup
+## Database setup
 
-**TiefVision** uses an **H2** database to store data regarding bounding boxes in images and also similarities between images.
-**TiefVision** expects a **H2** server deployed in the configuration parameter **slick.dbs.bounding_box.db.url** from **$TIEFVISION_HOME/src/scala/tiefvision-web/conf/application.conf**.
-The **H2** database must have the necessary tables created. The DDL SQL query that generates those tables can be found in **$TIEFVISION_HOME/src/scala/tiefvision-web/conf/ddl.sql**.
+**TiefVision** uses a database to store data regarding bounding boxes in images and also similarities between images.
+**TiefVision** has an easy to run **H2** docker image that can be built and run via **$TIEFVISION_HOME/src/h2/service**.
+**TiefVision**'s can be configured to use another database, like postgresql.
+Just change **slick.dbs.bounding_box.* ** from **$TIEFVISION_HOME/src/scala/tiefvision-web/conf/application.conf**.
```
Comment (paucarre) [inline doc/developer.md]: really nice you updated doc

**Review by paucarre — APPROVED** [review]
LGTM if DDL is in another place, which as far as I can remember you moved to create_tables.sql or something like that

File: src/scala/tiefvision-web/conf/ddl.sql (hunk 1 above)
Comment (Plippe) [inline src/scala/tiefvision-web/conf/ddl.sql]: It is, in the h2 folder

File: doc/developer.md (hunk 2 above)
Comment (Plippe) [inline doc/developer.md]: Better late than never :)

**Pull request merged**